roles/openshift_azure/tasks/remove_vm.yml

```diff
@@ -1,5 +1,10 @@
 ---
 - name: remove virtual machine
   azure_rm_virtualmachine:
+    resource_group: "{{ openshift_azure_resource_group_name }}"
+    name: "{{ openshift_azure_vm_name }}"
+    remove_on_absent:
+    - network_interfaces
+    - public_ips
+    - "{{ '' if openshift_azure_vm_keep_storage else 'virtual_storage' }}"
```

Comment:
This looks good.  Does this work if it is `''`?

Reply:
Yes, `''` doesn't matter (luckily).